src/gui/EditWidget.cpp

```diff
@@ -36,6 +36,7 @@ EditWidget::EditWidget(QWidget* parent)
     headerLabelFont.setBold(true);
     headerLabelFont.setPointSize(headerLabelFont.pointSize() + 2);
     headlineLabel()->setFont(headerLabelFont);
+    headlineLabel()->setTextFormat(Qt::RichText);
```

Comment:
Wrong way round.  Set text format to PlainText and remove HTML escaping in EditEntryWidget.cpp, lines 283-291.

We have more instances where we added HTML escaping (see #236). Thinking about this more, that was the wrong solution, though. Correct solution would be doing the above: use PlainText instead of explicit HTML escaping.